docs/dev/src/ci.md

```diff
@@ -0,0 +1,21 @@
+# CI
```

Comment:
```suggestion
# Continuous Integration
```

Design-wise, it looks weird just to have `CI`.

<img width="1478" alt="Screenshot 2024-07-02 at 12 49 56 PM" src="https://github.com/risingwavelabs/risingwave/assets/47273164/27eca43c-cccc-45e8-b24e-934567acb565">